### ISSUE TITLE
Support for serverless v3 variables resolver

### ISSUE THREAD

**Issue by wktk** [steveperkins/serverless-find-resource#4]
Hello, thanks for this useful software! 

I noticed that `serverless-find-resource` does not work with Serverless v3, as it has changed its way to handle variable resolvers incompatibly in v3.
https://www.serverless.com/framework/docs/deprecations#new-variables-resolver

When I run `sls deploy` and `sls doctor` I get the following output:
```sh
$ sls deploy
Environment: darwin, node 16.9.1, framework 3.2.1 (local), plugin 6.0.0, SDK 4.3.1
Credentials: Local, environment variables
Docs:        docs.serverless.com
Support:     forum.serverless.com
Bugs:        github.com/serverless/serverless/issues

Error:
Unrecognized configuration variable sources: ""

1 deprecation found: run 'serverless doctor' for more details

$ sls doctor
1 deprecation triggered in the last command:

Plugin "FindResourcePlugin" attempts to extend old variables resolver. Ensure to rely on latest version of a plugin and if this warning is still displayed please report the problem at plugin issue trackerStarting with next major release, old variables resolver will not be supported.

More info: https://serverless.com/framework/docs/deprecations/#NEW_VARIABLES_RESOLVER
```

(For reference, [this patch](https://github.com/wktk/serverless-find-resource/commit/4e42e5e9ca47e9bebf0d8717dd2c39011f2af31e) works fine for me.  As I am new to Serverless and not sure if the patch is correct, I'm not opening a pull request, but feel free to import it if you would like to.)

**Comment by a3linux** [comment]
@wktk Could you raise a PR here in this repository so that I can pull and test it. 
I think I have the same issue when my tool chain upgrade to sls 3.x. 

Thanks.

**Comment by steveperkins** [comment]
Merged PR #6 and published v1.0.11 to NPM.

**Comment by wktk** [comment]
Thank you!
